zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js

```diff
@@ -73,6 +75,18 @@ angular.module('zeppelinWebApp')
 
   };
 
+  $scope.renderText = function() {
+    var retryRenderText = function() {
+      if (angular.element('#' + $scope.paragraph.id + '_text').length) {
+        angular.element('#' + $scope.paragraph.id + '_text').perfectScrollbar();
+        var height = $scope.paragraph.config.graph.height;
+        angular.element('#p' + $scope.paragraph.id + '_text').height(height);
+      } else {
```

Comment:
I picked template function from $scope.renderHtml; fixed both $scope.renderHtml and $scope.renderText